charts/incrementals-publisher/values.yaml

```diff
@@ -50,12 +50,12 @@ affinity: {}
 permissions_url: ''
 artifactory:
   # artifactory.key -- key to upload to artifactory
-  key: ''
+  key: ""
```

Comment:
Is there a particular reason to change this? Could you keep the original simple quote under the proposed argument to stay coherent within the file?

Reply:
Just wanted to make it consistent across the file, but will rollback as other files are also having the same.

```yaml
nameOverride: ""
fullnameOverride: ""
```